src/Monitor/Monitor/Az.Monitor.psd1

```diff
@@ -133,7 +139,19 @@ PrivateData = @{
       - Remove-AzMetricAlertRuleV2
       - Get-AzMetricAlertRuleV2
       - Add-AzMetricAlertRuleV2
-  * Updated Monitor SDK to version 0.22.0-preview'
+		* Updated Monitor SDK to version 0.22.0-preview
```

Comment:
@shpimpal this change to the release notes can be reverted since we will update this section during our release